tests/charm++/within_node_bcast/within_node_bcast.ci

```diff
@@ -46,13 +49,65 @@ mainmodule within_node_bcast {
           }
         }
       }
+      serial { tests_completed++; }
+
+      serial { CkPrintf("Testing group broadcast\n"); }
+      for (test_num = 0; test_num < ITERATIONS; test_num++) {
+        serial {
+          CkCallback cb(CkReductionTarget(Main, testDone), thisProxy);
+          TestMessage* msg1 = new (DATA_SIZE) TestMessage(test_num, cb);
+          std::fill(msg1->data, msg1->data + DATA_SIZE, test_num);
+          msg1->counter.store(msg1->test_num);
+          ngProxy.recvCopyNoFwd(msg1);
+
+          TestMessage* msg2 = new (DATA_SIZE) TestMessage(test_num, cb);
+          std::fill(msg2->data, msg2->data + DATA_SIZE, test_num);
+          msg2->counter.store(msg2->test_num);
+          gProxy.recvCopy(msg2);
+        }
+
+        when testDone(int result1), testDone(int result2) serial {
+          if (result1 != result2) {
+            CkPrintf("Results for test %i do not match! (%i, %i)\n",
+                test_num, result1, result2);
+            CkAbort("Test failed in results checking!!\n");
```

Comment:
You can pass format string args to CkAbort directly now

Reply:
That's wonderful.